routers/web/web.go

```diff
@@ -150,6 +151,9 @@ func Routes(sessioner func(http.Handler) http.Handler) *web.Route {
 		rw.WriteHeader(200)
 	})
 
+	hc := &healthcheck.HealthChecker{}
+	routes.Get("/api/healthz", hc.Check)
```

Comment:
Why `healthz` and not just `health`?

Reply:
so I think we use `healthz` here is better, means it is not a real api. (but we mount it under `/api`)

Reply:
Okay, fine with me, I wasn't aware of this convention.